FirstApps/wishlist_app/lib/data/model.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+class Product {
+  final String title;
+  final String content;
+  bool isFavorited;
```

Comment:
Use `final`

If you want to change some attribute. Please implement `copyWith()`

Should implement == or use `equatable` package